.gitlab/tests/contrib.yml

```diff
@@ -130,6 +85,17 @@ jinja2:
   variables:
     SUITE_NAME: "jinja2"
 
+kafka:
```

Comment:
This file has a bunch of unrelated changes in it

Reply:
I'll revert the alphabetical changes.